Sources/web3swift/Tokens/ST20/Web3+ST20.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 import BigInt
-
+import Core
 
 // NPolymath Token Standard
```

Comment:
Please, change this comment to documentation and add the following link: https://github.com/PolymathNetwork/polymath-core/tree/master/docs#st-20-interface-overview